src/main/res/values/strings.xml

```diff
@@ -45,11 +45,11 @@
     <string name="has_security_vulnerabilities">Has security vulnerabilities</string>
     <string name="http_error_DESC">Invalid server response.</string>
     <string name="http_proxy">HTTP proxy</string>
-    <string name="ignore_all_updates">Ignore all updates</string>
-    <string name="ignore_this_update">Ignore this update</string>
+    <string name="ignore_all_updates">Ignore all new versions</string>
+    <string name="ignore_this_update">Ignore this version</string>
     <string name="incompatible_api_DESC_FORMAT">Your %1$s (API version %2$d) is not supported. %3$s</string>
-    <string name="incompatible_api_max_DESC_FORMAT">Maximum API version is %d.</string>
-    <string name="incompatible_api_min_DESC_FORMAT">Minimum API version is %d.</string>
+    <string name="incompatible_api_max_DESC_FORMAT">The max. API version is %d.</string>
+    <string name="incompatible_api_min_DESC_FORMAT">The min. API version is %d.</string>
```

Comment:
Having periods after max and min feels very wrong.

Reply:
Sorry, looked that up in the wrong language.